src/gui/qgsattributeform.cpp

```diff
@@ -418,6 +419,66 @@ bool QgsAttributeForm::saveEdits()
   return success;
 }
 
+bool QgsAttributeForm::updateDefaultValues( const int originIdx )
+{
+  if ( !mDefaultValueDependencies.contains( originIdx ) )
+    return false;
+
+  // create updated Feature
+  QgsFeature updatedFeature = QgsFeature( mFeature );
```

Comment:
won't we need QgsVectorLayerUtils::createFeature here?

Reply:
Won't that lead to a sudden increase for postgres serials with "enable default values server side" enabled?

Reply:
Quite possibly!